### ISSUE TITLE
launchctl doesn't work properly (as mobile) on 4k device with MS

### ISSUE THREAD

**Issue by blanxd** [pwn20wndstuff/Undecimus#706]
**Describe the bug**
_(title)_
This might be trivial, but making a report _just in case_. (well I use launchctl return codes in my SSH Toggle tweak so I noticed) (EDIT: and I'll update the tweak to work around it if necessary, no worries)

This isn't dependent on the daemon being queried, if it isn't running, launchctl returns 113 as it's supposed to, but if the daemon is running and launchctl should return 0, I get 154. This issue doesn't occur on my A11 device with u0-3.0.0-b41. And on 3.0.0-b40 launchctl worked correctly.

**To Reproduce**
```
[BlanxdH-iPhone5s:~ 13:00:14]$ launchctl list com.openssh.sshd
[BlanxdH-iPhone5s:~ 13:00:28]$ echo $?
154
[BlanxdH-iPhone5s:~ 13:00:32]$ launchctl error 154
154: Requestor is not a platform binary
[BlanxdH-iPhone5s:~ 13:00:42]$ sudo su -
[BlanxdH-iPhone5s:~ 13:00:48]# launchctl list com.openssh.sshd
{

... excerpt ...

};
[BlanxdH-iPhone5s:~ 13:00:50]# echo $?
0
[BlanxdH-iPhone5s:~ 13:00:55]# 
```

**Device:**
 - iOS Version: iOS 12.1.2 (16C104)
 - iOS Device: iPhone 5s (iPhone6,1)
 - unc0ver Version: 3.0.0~b41

**Comment by blanxd** [comment]
uuh seems it's the environment in general (shell or SpringBoard in this case). If I run it thru another bash (sh -c "launchctl list com.openssh.sshd") then it works, both CLI and GUI.

**Comment by blanxd** [comment]
fixed in 3.0.0~b42.
